changelogs/fragments/74241-find-checks-size-with-any.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fix a bug in the find module where 'size' argument was ignored for regular files with 'file_type' of 'any'.
```

```suggestion
  - find - Fix a bug where ``size`` argument was ignored for regular files with ``file_type`` of ``any``.
```

I generally try to put the module name at the start of changelogs, makes it easier for people to find at a glance.